new.go

```diff
@@ -351,7 +344,7 @@ func newBuilder(ctx context.Context, store storage.Store, options BuilderOptions
 	}
 
 	if options.Mount {
-		_, err = builder.Mount(mountLabel)
+		_, err = builder.Mount(container.MountLabel())
```

Comment:
Does the underlying `Store` handle this for us if we pass "" as the label?

Reply:
Yes